<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdministratorWidget</class>
 <widget class="QWidget" name="AdministratorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>911</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="placeholderText">
       <string>Введите Ваш запрос...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Исполнить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>911</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>911</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;«&lt;span style=&quot; font-weight:600;&quot;&gt;И от всякого, кому дано много, много и потребуется, и кому много вверено, с того больше взыщут&lt;/span&gt;»&lt;/p&gt;&lt;p&gt;Луки 12:48&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>AdministratorWidget</receiver>
   <slot>sendQuery()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>798</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>719</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sendQuery()</slot>
 </slots>
</ui>
